### DISCUSSION TITLE
Reload the Waveform after fileBlob or url is changed

### DISCUSSION THREAD

**Discussion by ssoronid** [katspaugh/wavesurfer.js#3341] [Q&A]
I would like to know whats the best practice in our usecase.

Users can upload files from local or they can be loaded via url (aws s3)

The problem if when a current.load happens it stays there even if the file is changed.

Is there a way to destroy and reload?

**Comment by katspaugh** [comment]
Calling load with a different URL should load and render that audio, replacing the previous one. If it doesn’t, it might be a bug.

Alternatively you can completely destroy the instance and create a new one like in this example: https://wavesurfer.xyz/examples/?react.js
